functional-tests/test-bed/src/test/java/org/arquillian/smart/testing/ftest/newtests/HistoricalChangesNewTestsSelectionExecutionFunctionalTest.java

```diff
@@ -47,7 +51,9 @@ public void should_only_execute_new_test_when_multiple_commits_with_changed_test
 
         // when
         final List<TestResult> actualTestResults = project
-            .withEnvVariables("git.last.commits", "3")
+            .buildOptions()
+                .withSystemProperties("git.last.commits", "3")
+                .configure()
```

Comment:
I probably don't understand your questions... By default, I'm trying to use the auto formatting only for changed part of the code. I just don't like when some part of the code has different formatting that it is set by default in our styles or `.editorconfig` file - then if I added a new code, then it would look different then your code...